src/Dynamo.All.2012.sln

```diff
@@ -209,7 +209,7 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CommandLineTests", "..\test
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DynamoPublish", "DynamoPublish\DynamoPublish.csproj", "{8EDA9F32-9D7D-4D55-9950-52FD695970A9}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DynamoPublishTests", "DynamoPublishTests\DynamoPublishTests.csproj", "{A6E8F8D6-A068-4E09-AAA6-7426729D56F6}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DynamoPublishTests", "..\test\DynamoPublishTests\DynamoPublishTests.csproj", "{A6E8F8D6-A068-4E09-AAA6-7426729D56F6}"
```

Comment:
I've noticed that I put `DynamoPublishTests` in `src` folder, which is incorrect. I moved it to `test` folder. By the way, `DynamoUtilitiesTests` also located in `scr` folder. If you would like, I can move it in `test` as well.